<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModulesView</class>
 <widget class="QWidget" name="ModulesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>36</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Modules</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mod1Button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>180</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>What Causes Avalanches?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mod2Button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>250</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Types of Avalanches</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mod3Button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>320</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>What you can do</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>190</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>40</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>478</x>
     <y>253</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>40</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>40</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="certificateButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>400</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>View Completion Page</string>
   </property>
  </widget>
  <widget class="QLabel" name="progress">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>You have completed 0 / 3 modules.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
